src/eq_schema/schema/Questionnaire/index.js

```diff
@@ -0,0 +1,127 @@
+const { last } = require("lodash");
+
+const { SOCIAL } = require("../../../constants/questionnaireTypes");
+const {
+  types: { VOLUNTARY },
+  contentMap
+} = require("../../../constants/legalBases");
+
+const { Confirmation, Introduction, Summary } = require("../../block-types");
+
+const Section = require("../Section");
+const Hub = require("../Hub");
+
+const DEFAULT_METADATA = [
+  {
+    name: "user_id",
+    type: "string"
+  },
+  {
+    name: "period_id",
+    type: "string"
+  },
+  {
+    name: "ru_name",
+    type: "string"
+  }
+];
+
+const SOCIAL_THEME = "social";
+const DEFAULT_THEME = "default";
+const NI_THEME = "northernireland";
+
+const DEFAULT_METADATA_NAMES = DEFAULT_METADATA.map(({ name }) => name);
+
+class Questionnaire {
+  constructor(questionnaireJson) {
+    const questionnaireId = questionnaireJson.id;
+    this.eq_id = questionnaireId;
+    this.form_type = questionnaireId;
+    this.mime_type = "application/json/ons/eq";
+    this.schema_version = "0.0.1";
+    this.data_version = "0.0.3";
+    this.survey_id =
+      questionnaireJson.surveyId ||
```

Comment:
Survey ID is not on the root level of the questionnaire Author JSON, so it'll only ever add in the title. This should be grabbed from the `publishDetails` object that can be found at the root level of the questionnaire object.